samples/ngIdentity/README.md

```diff
@@ -0,0 +1,114 @@
+
+1. Added the `Microsoft.AspNetCore.Identity.EntityFrameworkCore` NuGet package.
+1. Added the `Microsoft.EntityFrameworkCore.InMemory` NuGet package. You can use any relational EF Core provider, but you will need to properly create/migrate the database (see [Migrations Overview](https://learn.microsoft.com/ef/core/managing-schemas/migrations/?tabs=dotnet-core-cli)).
+1. Added the identity user and database.
+   ```csharp
+   class MyUser : IdentityUser { }
+
+   class AppDbContext(DbContextOptions<AppDbContext> options) :
+      IdentityDbContext<MyUser>(options)
+   {    
+   } 
+   ```
+1. Configured the identity services. The code here changes the default behavior of returning a `404` when not authenticated to a `401` so the client can automatically redirect to the login page.
+   ```csharp
+   builder.Services.AddAuthentication(IdentityConstants.ApplicationScheme)
+    .AddIdentityCookies()
+    .ApplicationCookie!.Configure(opt => opt.Events = new CookieAuthenticationEvents()
+    {
+        OnRedirectToLogin = ctx =>
+        {
+            ctx.Response.StatusCode = 401;
+            return Task.CompletedTask;
+        }
+    }); 
+    builder.Services.AddAuthorizationBuilder();
+   ```
+1. Configured the EF Core `DbContext` to use the in-memory identity database, and turned on the identity APIs.
+   ```csharp
+   builder.Services.AddDbContext<AppDbContext>(
+      options => options.UseInMemoryDatabase("AppDb"));
+
+   builder.Services.AddIdentityCore<MyUser>()
+     .AddEntityFrameworkStores<AppDbContext>()
+     .AddApiEndpoints();
+   ```
+1. Mapped the identity API endpoints: `app.MapIdentityApi<MyUser>();`
+1. Added a logout method. This isn't included in the endpoints because it may change based on your configuration (i.e. cookies, tokens, both). More on this method below.
+1. Locked down the weather forecast API to only respond to authenticated users by adding the extension: `.RequireAuthorization();`
+
+The `logout` endpoint is defined like this:
+
+```csharp
+app.MapPost("/logout", async (
+    SignInManager<MyUser> signInManager,
+    [FromBody]object empty) =>
+{
+    if (empty is not null)
+    {
+        await signInManager.SignOutAsync();
+        return Results.Ok();
+    }
+    return Results.NotFound();
+}).RequireAuthorization();
+```
+
+Minimal APIs allow you to bind to forms. When you bind to forms, you risk exposing your site to cross-site request forgery (CSRF) attacks. To prevent this, you use the [anti-forgery services](https://learn.microsoft.com/aspnet/core/security/anti-request-forgery?view=aspnetcore-8.0). This logout doesn't bind to a form, but it also takes no parameters. Without parameters, an empty forms post from a malicious site could invoke the API. To avoid this, the client will post with an empty body: `{}`. The server checks for the parameter. If the empty body is included, the `empty` parameter is not null (it is deserialized as a `JObject`) and the method will succeed.
```

Comment:
```suggestion
Minimal APIs can bind to forms. When binding to forms, the site is at risk of cross-site request forgery (CSRF) attacks. To prevent CSRF attacks, use the [anti-forgery services](https://learn.microsoft.com/aspnet/core/security/anti-request-forgery). This logout doesn't bind to a form, but it also takes no parameters. Without parameters, an empty forms post from a malicious site could invoke the API. To avoid this, the client posts with an empty body: `{}`. The server checks for the parameter. If the empty body is included, the `empty` parameter is not `null` and the method succeeds. The `empty` parameter is deserialized as a `JObject`.
```